Block/RecentCommentsBlockService.php

```diff
@@ -72,9 +82,9 @@ public function execute(BlockContextInterface $blockContext, Response $response
     /**
      * {@inheritdoc}
      */
-    public function validateBlock(ErrorElement $errorElement, BlockInterface $block)
```

Comment:
Beause it' inherited from https://github.com/sonata-project/SonataBlockBundle/blob/master/Block/BaseBlockService.php#L198